docs/contributor/greenboot.md

```diff
@@ -23,9 +23,9 @@ at this time.
 
 Proceed by creating a health check script in the `/etc/greenboot/check/required.d`
 directory.
-> The name prefix of the user script should be chosen to make sure it runs after
-> the `40_microshift_running_check.sh` script, which implements the MicroShift
-> health check procedure for its core services.
+> The name prefix of the user script should be chosen appropriately.
+> Because MicroShift's healthcheck resides in `/var/lib/greenboot/check/require.d`,
```

Comment:
Should this be /usr/lib instead of /var/lib?

Reply:
Yes!